tests/bluetooth/bsim/host/adv/encrypted/src/central.c

```diff
@@ -0,0 +1,125 @@
+/* Copyright (c) 2023 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+
+#include <zephyr/logging/log.h>
+#include <zephyr/bluetooth/addr.h>
+#include <zephyr/bluetooth/bluetooth.h>
+
+#include <encrypted_ad_data.h>
+
+#include "bs_types.h"
+#include "bs_tracing.h"
+#include "bstests.h"
+#include "zephyr/toolchain/gcc.h"
+
```

Comment:
All these common defines and data could be moved to a common header so that they dont have to be manually be kept in sync between the two files.